setup.py

```diff
@@ -234,14 +234,13 @@
 import platform
 
 
-BUILD_LIBTORCH_WHL = os.getenv("BUILD_LIBTORCH_WHL", "0") == "1"
-BUILD_PYTHON_ONLY = os.getenv("BUILD_PYTHON_ONLY", "0") == "1"
-
+# Also update `project.requires-python` in pyproject.toml when changing this
```

Comment:
It would be great if we could detect build isolation and raise a warning

Reply:
I don't think it's possible, nor is it desirable - it's perfectly valid for example for a packager to build with `python -m build --wheel`.